extensions/wikia/WallNotifications/scripts/WallNotifications.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * This file is handling WallNotifications on MonoBook
+ */
 var $window = $(window);
 var WallNotifications = $.createClass(Object, {
 	constructor: function() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)